### ISSUE TITLE
Should "downloads over time" graphs be reduced to past few months? (and general graphs gripes)

### ISSUE THREAD

**Issue by barthalion** [flathub/frontend#207]
1. For applications that are available on Flathub for a long time, the graph does not seem very useful right now, as there were fewer downloads in the past than now. As these numbers aren't very indicative of anything (as we don't track machine-id or IPs), would it make sense to limit them to past few months, 3, 6 or 12?

![image](https://user-images.githubusercontent.com/380403/153628820-4b81012b-d4e2-4d2a-ac35-ed4bff1ee8ee.png)

2. The spikes in the graph above are most likely caused by updates. Should we graph only downloads with updates subtracted, to flatten it a bit? Alternatively, should we apply some "weight" to make the graph appear less suspicious?

3. In the past, we haven't agreed on displaying graphs at all to avoid disappointing new developers that their applications aren't very popular (more or less). Not sure if that's something we want to stick to for new submissions.

cc @nedrichards

**Comment by barthalion** [comment]
Similar question about the general usefulness of the downloads tile.

**Comment by gasinvein** [comment]
> The spikes in the graph above are most likely caused by updates. Should we graph only downloads with updates subtracted, to flatten it a bit?

I believe we should. These graphs are seen as app popularity meters, so update frequency shouldn't contribute to it.

**Comment by gasinvein** [comment]
Installs-only graph looks much better IMO.

**Comment by razzeee** [comment]
Yeah, the explore page also seems more reasonable with it https://github.com/flathub/backend/pull/116

**Comment by nedrichards** [comment]
I think we're not necessarily thinking of things from a developers or users perspective here about what we're trying to communicate with all of this.

'I want to know if this app is popular'
'I want to show off that a lot of people use my app'
'I want to know how many active users I have on flathub'

For the first one the 'installs only' metric makes good sense for the reasons you mention in https://github.com/flathub/backend/pull/116 although if you're looking at a specific item page like we're talking about here I think there's a case to be made for a 'bigger number' - that might be better expressed as a rounded, tiered 'peak number from the last 3 months' though (e.g. 25,000+ downloads) rather than a graph.

For the second one the developer just wants to show the absolute biggest number they can - in the 'graph off' above they'd prefer you to stick a line of best fit through the ugly one and show the higher numbers. Again, they might be happier with a 'show the biggest number in the last $window'.

The latter one can only be a unified combination and they probably do want to see both kinds of graphs and maybe get a nice export to CSV or similar. The best place for that is obviously inside the developer portal part of the site rather than on the actual item page!

Can anyone else think of some use cases I might have missed?
